src/groups/mwc/mwcc/mwcc_orderedhashmap.t.cpp

```diff
@@ -826,7 +826,7 @@ static void test14_localIterator()
 
     const size_t bucketCount = map.bucket_count();
 
-    int key         = bucketCount / 2;
+    int key         = static_cast<int>(bucketCount / 2);
```

Comment:
I wonder if the right approach here is changing `MyMapType` to be an `mwcc::OrderedHashMap<size_t, size_t, IdentityHasher>`.  I think this will also need a change in `IdentityHasher`.  This is a case like the one above, where I think really what we would want is C++11 `auto`---if only we were able to use C++11...  So changing `int`s to `size_t`s might be the next best thing.

Good intuition here for going with the solution that is least invasive.  Here we're in a test file, though, so we don't need to worry about breaking interfaces or even changing behavior subtly---we just want to make sure that we still test all the cases we currently test.  This means we can be a little more aggressive in this file doing the morally right changes (fixing types, avoiding casts).

Reply:
Sounds good, fixing the data types seems to be a better code practice approach here.

I'll test out your suggested changes (basically see if we can get away with fixing more proper datatypes without conflicts with other tests and implementations) soon tonight to see if they work properly (and pass the CI checks).

Reply:
Excellent, thanks!